client/src/components/MapComponent/index.js

```diff
@@ -38,6 +38,16 @@ const MapComponent = ({ mapInfo }) => {
       zoom,
     });
     new mapboxgl.Marker().setLngLat([long, lat]).addTo(map.current);
+    map.current.addControl(new mapboxgl.NavigationControl());
+    map.current.addControl(
+      new mapboxgl.GeolocateControl({
+        positionOptions: {
+          enableHighAccuracy: true,
+        },
+        trackUserLocation: true,
+      })
+    );
+    map.current.scrollZoom.disable();
```

Comment:
Instead of this add `interactive: false,`  to the options since there's still a problem with the smartphones

```js
    map.current = new mapboxgl.Map({
      container: mapContainer.current,
      style: 'mapbox://styles/mapbox/streets-v11',
      center: [long, lat],
      zoom,
      interactive: false,
    });
```

Reply:
Done